pages/a_la_francaise.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="fr">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta
+            name="description"
+            content="Ohmyfood le menu à la française"
+        />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>À la française Ohmyfood</title>
```

Comment:
Please pu a - or a | between menu name and ohmyfood

Reply:
i put this "|" between menu name and brand name